CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to irida-galaxy-importer will be documeted in this file.
 
+## 2.1.0
+* Added in support for importing IRIDA files that are not available locally (i.e. in the cloud)
+
+## 2.0.1
```

Comment:
We never had a proper release for 2.0.1 so this should all be under 2.1.0

Reply:
Fixed in bb491c8